server/gokbg3/grails-app/controllers/org/gokb/PublicController.groovy

```diff
@@ -102,7 +104,14 @@ class PublicController {
       mutableParams.tempFQ = ' -pkg_scope:\"Master File\" -\"open access\" ';
 
     result =  ESSearchService.search(mutableParams)
-    result.transforms = grailsApplication.config.packageTransforms
+
+    Calendar calendar = Calendar.getInstance()
+    result.componentsOfStatistic = ["TitleInstance", "Org", "Package", "Platform", "CuratoryGroup", "TitleInstancePackagePlatform"]
+
+    result.countComponent = [:]
+    result.componentsOfStatistic.each {String component ->
+      result.countComponent."${component}" = ComponentStatistic.executeQuery("select numTotal from ComponentStatistic where componentType = ? and year = ? and month = ?", [component, calendar.get(Calendar.YEAR), calendar.get(Calendar.MONTH)], [readOnly: true])[0]
```

Comment:
Using ? as queryParam is actually deprecated and against LAS:eR coding convention